docs/data-sources/waf_rules.md

```diff
@@ -145,4 +141,27 @@ The `rules` block supports:
 * `latest_revision_number` - The rule's latest revision.
 * `type` - The rule's type.
 
-[1]: https://docs.fastly.com/api/ngwaf#ngwaf_rules
+[1]: https://developer.fastly.com/reference/api/waf/rules/
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- **exclude_modsec_rule_ids** (List of Number) A list of modsecurity rules IDs to be excluded from the data set.
+- **id** (String) The ID of this resource.
+- **publishers** (List of String) A list of publishers to be used as filters for the data set.
+- **tags** (List of String) A list of tags to be used as filters for the data set.
+
+### Read-only
+
+- **rules** (List of Object) The list of rules that results from any given combination of filters. (see [below for nested schema](#nestedatt--rules))
+
+<a id="nestedatt--rules"></a>
+### Nested Schema for `rules`
+
+Read-only:
+
+- **latest_revision_number** (Number)
```

Comment:
There are a few issues open already with the `tfplugindocs` tool only just being release. I've opened an issue related to nested Schema 'Description' fields not being included in the output.

https://github.com/hashicorp/terraform-plugin-docs/issues/28